exercise2/exercise8.js

```diff
@@ -0,0 +1,18 @@
+/*
+function discount(totalamount) {
+  var disc;
+  if (totalamount > 100) {
+    disc = totalamount * 0.1;
+  } else {
+    disc = totalamount * 0.05;
+  }
+  return disc;
+}
+*/
+function discount(totalAmount) {
+
+    if (totalAmount > 100) {
+        return totalAmount * 0.1;
+    }
+    return totalAmount * 0.05;
```

Comment:
a ternary operator here would make it a bit nicer


```suggestion
    return totalAmount > 100 ? totalAmount * 0.1 : totalAmount * 0.05;
```